views/js/previewer/plugins/content/scss/highlighterTray.scss

```diff
@@ -0,0 +1,71 @@
+.highlighter-container {
+    --highlighter-tray-width: 30rem;
+    --color-button-size: 16px;
+    --color-button-hover-size: 24px;
+
+    position: absolute;
+    top: 0.75rem;
+    left: calc(100% - var(--highlighter-tray-width) - 3rem);
+    z-index: 1;
+
+    & .highlighter-tray {
+        border-radius: 8px;
+        padding: 1rem;
+        width: var(--highlighter-tray-width);
+        height: 4.5rem;
+        margin-right: 30px;
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        background-color: #1F1F1F;
+
+        & .highlighter-title {
+            text-transform: uppercase;
+            color: white;
+            border: none;
+            margin-right: 1rem;
+        }
+
+        & .icon-eraser {
+            color: white;
+            background: transparent;
+        }
+
+        & .color-item {
+            flex-basis: 33.3%;
+            display: flex;
+            justify-content: center;
+        }
+
+        & .color-button {
+            border-radius: var(--color-button-size);
+            height: var(--color-button-size);
+            width: var(--color-button-size);
+            min-width: var(--color-button-size);
+            border: transparent;
+            cursor: pointer;
+
+            &:hover {
+                border-radius: var(--color-button-hover-size);
+                height: var(--color-button-hover-size);
+                width: var(--color-button-hover-size);
+            }
+
+            &:focus {
+                border: transparent !important;
```

Comment:
Otherwise for me this style is overwritten by this rule:
https://github.com/oat-sa/tao-core-ui-fe/blob/0c2ad82591b6799debde9c2f6403e1ddd0223596/scss/inc/_forms.scss#L6

Reply:
Damn! One other legacy sh...
Good catch!